netbox/netbox/constants.py

```diff
@@ -15,4 +15,14 @@
     'available-ips': 100200,
     'available-vlans': 100300,
     'available-asns': 100400,
+
+    # MPTT locks
+    'region': 100500,
```

Comment:
Should we carve out a separate number space for MPTT locks? We might want to add more "available object" endpoints in the future and it would be cleaner to keep those numbers consistent.

Reply:
That's why I put them all under 1005xx, if we add more available they can be at 100600, 100700, etc.. We might add more MPTT locks in the future so this allows them to easily co-exist.

Reply:
Right; I'm saying maybe we should reserve 100xxx for views and use e.g. 101xxx for MPTT models. It's not  big deal, just trying to plan for future additions.